temporalcli/commands.workflow_view_test.go

```diff
@@ -296,18 +296,24 @@ func (s *SharedServerSuite) testWorkflowShowFollow(eventDetails bool) {
 	res := <-outputCh
 	s.NoError(res.Err)
 	output := res.Stdout.String()
-	if eventDetails {
-		s.Contains(output, "my-signal")
-	}
+	// Confirm result present
 	s.ContainsOnSameLine(output, "Result", `"hi!"`)
 	s.NoError(run.Get(s.Context, nil))
+
+	// Detailed uses sections, non-detailed uses table
+	if detailed {
+		s.Contains(output, "input[0]: ignored")
+		s.Contains(output, "signalName: my-signal")
+	} else {
+		s.Contains(output, "WorkflowExecutionSignaled")
+	}
```

Comment:
This would show up in both output kinds, right? Non-detailed on the line, and detailed in the section header.

Reply:
Yes, but before this the test tested nothing about non-detailed output, so I just thought I'd add it in an `else` for code clarity reasons (I can do better assertions on detailed). But I can move it out of else.

Reply:
Yeah I'd just assert it either way